I can't seem to get this branch to build, error is

```
manual-mkdocs  | Traceback (most recent call last):
manual-mkdocs  |   File "/usr/local/bin/mkdocs", line 8, in <module>
manual-mkdocs  |     sys.exit(cli())
manual-mkdocs  |              ^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1485, in __call__
manual-mkdocs  |     return self.main(*args, **kwargs)
manual-mkdocs  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1406, in main
manual-mkdocs  |     rv = self.invoke(ctx)
manual-mkdocs  |          ^^^^^^^^^^^^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1873, in invoke
manual-mkdocs  |     return _process_result(sub_ctx.command.invoke(sub_ctx))
manual-mkdocs  |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 1269, in invoke
manual-mkdocs  |     return ctx.invoke(self.callback, **ctx.params)
manual-mkdocs  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/click/core.py", line 824, in invoke
manual-mkdocs  |     return callback(*args, **kwargs)
manual-mkdocs  |            ^^^^^^^^^^^^^^^^^^^^^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/mkdocs/__main__.py", line 272, in serve_command
manual-mkdocs  |     serve.serve(**kwargs)
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/mkdocs/commands/serve.py", line 85, in serve
manual-mkdocs  |     builder(config)
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/mkdocs/commands/serve.py", line 67, in builder
manual-mkdocs  |     build(config, serve_url=None if is_clean else serve_url, dirty=is_dirty)
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/mkdocs/commands/build.py", line 299, in build
manual-mkdocs  |     nav = config.plugins.on_nav(nav, config=config, files=files)
manual-mkdocs  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/mkdocs/plugins.py", line 596, in on_nav
manual-mkdocs  |     return self.run_event('nav', nav, config=config, files=files)
manual-mkdocs  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
manual-mkdocs  |   File "/usr/local/lib/python3.12/site-packages/mkdocs/plugins.py", line 566, in run_event
manual-mkdocs  |     result = method(item, **kwargs)
manual-mkdocs  |              ^^^^^^^^^^^^^^^^^^^^^^
manual-mkdocs  | TypeError: on_nav() got an unexpected keyword argument 'config'
```

Not sure if this is related, but an interesting FYI about the future of `mkdocs` and `mkdocs-material`: https://squidfunk.github.io/mkdocs-material/blog/2026/02/18/mkdocs-2.0/

--- a/hooks/heading_numbers.py
+++ b/hooks/heading_numbers.py
@@ -94,7 +94,7 @@ def on_config(config):
     return config
 
 
-def on_nav(nav, _config, _files):
+def on_nav(nav, **kwargs):
     """Prefix sidebar section titles with their chapter/appendix number."""
 
     def _prefix(items):

